bot_impl.py

```diff
@@ -3886,14 +3890,22 @@ async def on_message(message):
 
                 vote = global_vars.game.days[-1].votes[-1]
 
+                voting_player = (await get_player(message.author))
+                if not voting_player:
+                    await safe_send(global_vars.channel, "You are not a player in the game.")
+                    return
                 if (
                     vote.order[vote.position].user
-                    != (await get_player(message.author)).user
+                    != voting_player.user
                 ):
                     await safe_send(global_vars.channel, "It's not your vote right now.")
                     return
 
                 vt = int(argument == "yes" or argument == "y")
+                voudon_in_play = next((x for x in global_vars.game.seatingOrder if x.character.role_name == "Voudon" and not x.is_ghost),None)
```

Comment:
here as well.  You might also want to consider pulling this out into a helper function since you're doing this 3 times now.